presto-docs/src/main/sphinx/installation/cli.rst

```diff
@@ -14,7 +14,90 @@ make it executable with ``chmod +x``, then run it:
 
     ./presto --server localhost:8080 --catalog hive --schema default
 
-Run the CLI with the ``--help`` option to see the available options.
+The available options for the CLI are:
```

Comment:
I don't like doing this. We're not providing any additional value here in the manual and it's something that will become out of date quickly. Better to have them run ``--help`` themselves and see the correct help.